config/environments/production.rb

```diff
@@ -34,37 +34,31 @@
                          [:memory_store, { size: 64.megabytes }]
                        end
 
-  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
-  # Requires an encryption key in `ENV['RAILS_MASTER_KEY']` or
-  # `config/secrets.yml.key`.
-  config.read_encrypted_secrets = true
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # config.require_master_key = true
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = Uglifier.new(harmony: true)
+  # Compress CSS using a preprocessor.
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
   config.assets.compile = false
 
-  # Allow the browser to cache images.
-  config.serve_static_assets = true
```

Comment:
🤷‍♂️

Reply:
This may be necessary for Heroku.

Reply:
Or maybe it's rolled up into somewhere else in Rails 6.

Reply:
@veganstraightedge so apparently this has nothing to do with assets: `https://github.com/rails/rails/commit/463b5d7581ee16bfaddf34ca349b7d1b5878097c`